.travis.yml

```diff
@@ -8,6 +8,7 @@ rvm:
   - 2.3.0
   - 2.4.1
   - 2.5.3
+  - 2.6.3
```

Comment:
[Ruby 2.6.5 was released](https://www.ruby-lang.org/en/news/2019/10/01/ruby-2-6-5-released/). I think would be better to test with the last stable.

Reply:
Ruby versions 2.4.9, 2.5.7, 2.6.5 have been released. Details at https://www.ruby-lang.org/en/

Reply:
You could also just to 2.6 since rvm aliars 2.6 to 2.6.latest. I did it for fx https://github.com/teoljungberg/fx/commit/d9c24fc148faba6cd1692e83f2774442c3978950